roles/aap_rules_validation/tasks/check_credentials_encryption.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Init sensitive data list
+  ansible.builtin.set_fact:
+    __sensitive_data: []
+    __unencrypted: []
+
+- name: Set sensitive list
+  ansible.builtin.set_fact:
+    __sensitive_data: "{{ __sensitive_data + ([item.key] | product(item.value)) }}"
+  with_dict: "{{ rule['credential_sensitive_data'] }}"
```

Comment:
The `with_dict` is considered legacy. Should use `loop: "{{ rule['credential_sensitive_data'] | dict2items }}"` instead.

See https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#with-dict

Reply:
thanks !
done in [980ea8a3](https://github.com/redhat-cop/aap_configuration_extended/pull/16/commits/980ea8a3d4b56e441ef635ab4d623bce0b59ad3d)